examples/asr/conf/config.yaml

```diff
@@ -7,7 +7,7 @@ labels: &labels [" ", "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l"
          "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "'"]
 
 model:
-  pl: null
+  cls: ???
```

Comment:
What's this for?

Reply:
It fixes an error introduced in #859

Reply:
Ah in fact that needs to be patched asap. We should make no assumption that cls exists or that user has to pass it explicitly since it's highly verbose and wrote prone.

Reply:
I mean technically, there is no one "cls" for the entire Model - maybe the entire ModelPT but that's harsh to require

Reply:
@blisc instead of ??? could you make it null temporarily until @okuchaiev reviews and merges my patch?